src/index.js

```diff
@@ -205,7 +193,7 @@ export default function undoable (reducer, rawConfig = {}) {
     initialState: rawConfig.initialState,
     initTypes: parseActions(rawConfig.initTypes, ['@@redux/INIT', '@@INIT']),
     limit: rawConfig.limit,
-    filter: rawConfig.filter || () => true,
+    filter: rawConfig.filter || (() => true),
```

Comment:
My apologies, the parenths are indeed needed. I duplicated some of YingshanDeng's efforts, by manually removing history in src and test of version 0.6.0. My changes were very similar, except for significant differences in the package.json file.